libmachine/persist/filestore.go

```diff
@@ -100,14 +100,6 @@ func (s Filestore) Exists(name string) (bool, error) {
 	return false, err
 }
 
-func (s Filestore) loadConfig(name string) (*host.Host, error) {
-	data, err := ioutil.ReadFile(filepath.Join(s.GetMachinesDir(), name, "config.json"))
```

Comment:
The `filestore.go` changes seem unrelated to the ` Declare the variable where it is used ` commit.

Reply:
done!